spec/features/create_a_user_spec.rb

```diff
@@ -1,8 +1,15 @@
 require 'rails_helper'
 
-RSpec.feature 'Create a User' do
-  scenario 'They click a button on the homepage that lets them sign up' do
-    visit  '/'
-    click_button 'Sign Up'
+RSpec.feature 'Create a User', :type => :feature do
+  scenario 'New users can sign up' do
+      visit '/'
+      click_link 'Sign Up'
+      expect(page).to have_content 'Sign Up Here'
```

Comment:
Can you do this while preserving the order of the [4 test phases](https://thoughtbot.com/blog/four-phase-test)? If you? 

Typically, feature specs assert that buttons and links work as intended. You don't need to assert that static text is on the page, especially when there are unique buttons that you can use to verify where you are in the next step.

Reply:
Good point. I removed the assertions of static checks and also added an assertion that the user is saved. What do you think? 
[https://github.com/orhanna14/homebrew-buddy/pull/2/commits/58dd7e55397d5bebefa2aaec03dede8afbef4acc](url)